src/Compilers/CSharp/Portable/BoundTree/Formatting.cs

```diff
@@ -148,9 +150,7 @@ public override object Display
     internal partial class BoundStackAllocArrayCreation
     {
         public override object Display
-        {
-            get { return string.Format(MessageID.IDS_StackAllocExpression.Localize().ToString(), ElementType, Count.Syntax); }
```

Comment:
>IDS_StackAllocExpression [](start = 49, length = 24)

It doesn't look like there is anything localizable in this resource - "stackalloc {0}[{1}]". I think we should remove it from resource file and hardcode the format string here. Then it becomes the same problem as for BoundTupleExpression. Note, it is fine to call ToString() on Count.Syntax and we should do this.